docs/contributor/05-10-e2e_tests.md

```diff
@@ -142,13 +146,16 @@ The test executes the following steps:
 The test executes the following steps:
 
 1. Provisions a Kyma runtime cluster.
-2. Creates a binding using Kubernetes TokenRequest and saves the returned kubeconfig.
+2. Creates a SKR binding and saves the returned kubeconfig.
 3. Initializes a Kubernetes client with the returned kubeconfig.
-4. Tries to fetch a Secret using the binding from Kubernetes TokenRequest.
-5. Creates a binding using Gardener and saves the returned kubeconfig.
-6. Initializes a Kubernetes client with the returned kubeconfig.
-7. Tries to fetch a Secret using the binding from Gardener.
-8. Deprovisions the Kyma runtime instance and cleans up the resources.
+4. Fetches a Secret using the SKR binding.
+5. Fetches created SKR binding.
+6. Deletes created SKR binding.
+7. Tries to fetch Secret using the deleted SKR binding.
```

Comment:
a Secret/the Secret?

Reply:
It's `sap-btp-manager` secret. Should I change it to: Tries to fetch the `sap-btp-manager` secret using the deleted Kyma Binding?